packages/commonwealth/client/scripts/views/pages/Editor/Editor.scss

```diff
@@ -0,0 +1,99 @@
+html,
+body,
+#root {
+  max-height: 100vh;
+  overflow: auto;
+}
+
+#root {
+  // this is needed for mobile layout
+  display: flex;
+}
+
+// disable misc buttons on mobile for now.
+#mava,
+.ReactQueryDevtools * {
+  display: none;
+}
+
+.mdxeditor {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+
+  // this is needed so that we can have a progress indicator on top of the main
+  // content.
+  position: relative;
+
+  .mdxeditor-toolbar {
+    border-top: solid #c6c6c6 1px;
+    border-radius: inherit;
+    background-color: inherit;
+    overflow: hidden;
+
+    position: initial;
+    top: initial;
+    height: 65px;
+  }
+}
+
+.mdxeditor-diff-source-wrapper {
+  overflow: auto;
+}
+
+.mdxeditor-diff-source-wrapper,
+.mdxeditor-root-contenteditable,
+.mdxeditor-rich-text-editor {
+  display: flex !important;
+  flex-grow: 1;
+}
+
+.mdxeditor-block-type-select,
+.mdxeditor-block-type-select button {
+  width: 45px;
+}
+
+.mdxeditor-container {
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+}
+
+.mdxeditor-root-contenteditable {
+  p {
+    line-height: 1.8em;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    line-height: 2.2em;
+  }
+
+  blockquote {
+    margin-left: 20px;
+    border-left: #c6c6c6 solid 3px;
+    padding-left: 5px;
+  }
+
+  ol,
+  ul {
+    margin-left: 20px;
+  }
+
+  ol {
+    list-style: decimal;
+  }
+
+  ul {
+    list-style: square;
+  }
+
+  //* {
```

Comment:
remove

Reply:
Actually this is something that I wanted to talk about.

Thanks for pointing it out.

We have a normalize.css script that we're using that clears the list-style and other default browser settings.

This has to be there or the list items, and other things don't work :-/

Reply:
this code is commented out so how this supposed to be working? I am fine with the code above, just talking about commented out